components/pricing/ForCollectiveCard.js

```diff
@@ -181,8 +176,18 @@ const ForCollectiveCard = () => {
               <FeeDescription>
                 <FormattedMessage
                   id="pricing.forCollective.hostFees"
-                  defaultMessage="{fee} apply depending on your host"
-                  values={{ fee: <FeeData whiteSpace="nowrap">Host fees</FeeData> }}
+                  defaultMessage="{hostFees} may apply depending on your <FiscalHostLink>Fiscal Host</FiscalHostLink>"
+                  values={{
+                    hostFees: (
+                      <FeeDataNoWrap>
+                        <FormattedMessage defaultMessage="Host fees" />
+                      </FeeDataNoWrap>
+                    ),
```

Comment:
@SudharakaP I fixed this in https://github.com/opencollective/opencollective-frontend/pull/7177/commits/6074753bde3ae3b4105f2db1e9ffd9f2e7cb1164#diff-38905975499d2bd9ae896f6944310e2aa537c8878def9c31b8b7acdf321f0d84R181. Why revert it? The tag version is way easier for contributors to translate: how does one know what `{hostFees}` will be replaced with (a number? a string?). And what about the word orders?

```suggestion
                  defaultMessage="<Fee>Host fees</Fee> may apply depending on your <FiscalHostLink>Fiscal Host</FiscalHostLink>"
                  values={{
                    hostFees: FeeDataNoWrap,
```

Reply:
Ops sorry, actually this one I was doing some refactoring to check something and forgot to revert before checkin. Sorry about that. Yes your fix is the correct on. I've put it back.